### PR TITLE
(PC-23131) feat(Venue): add dynamic socials on venue

### PR DESCRIPTION
**Pull request by marineshaw**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-23131

## Checklist

I have:

- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

## Screenshots

**delete** *if no UI change*

| Platform         | Before | After |
| :--------------- | :----: | :---: |
| iOS              |        |image image |
| Android          |        |       |
| Phone - Chrome   |        |       |
| Tablet - Chrome  |        |       |
| Desktop - Chrome |        |       |
| Desktop - Safari |        |       |

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/internal/cheatcodes/pages/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -205,12 +205,6 @@ export function Navigation(): React.JSX.Element {
               onPress={() => navigate('RecreditBirthdayNotification')}
             />
           </Row>
-          <Row half>
-            <ButtonPrimary
-              wording="🦔 Spike réseaux sociaux dynamiques"
-              onPress={() => navigate('DynamicSocials')}
-            />
-          </Row>
```
Comment (Lucasbeneston) [inline src/features/internal/cheatcodes/pages/Navigation/Navigation.tsx:213]: Bien 👍

File: __snapshots__/features/venue/components/VenueBody/VenueBody.native.test.tsx.native-snap [hunk 2]
```diff
@@ -2823,6 +2823,511 @@ exports[`<VenueBody /> should render correctly 1`] = `
         />
       </View>
     </View>
+    <View>
```
Comment (Lucasbeneston) [inline __snapshots__/features/venue/components/VenueBody/VenueBody.native.test.tsx.native-snap:2826]: Pourquoi nous avons un snapshot sur VenueBody et Venue la page ? :(

**Comment by Lucasbeneston** [comment]
> SonarCloud Quality Gate failed.    [Quality Gate failed](https://sonarcloud.io/dashboard?id=pass-culture_pass-culture-app-native&pullRequest=5050)
> 
> [Bug](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=CODE_SMELL) [2 Code Smells](https://sonarcloud.io/project/issues?id=pass-culture_pass-culture-app-native&pullRequest=5050&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=pass-culture_pass-culture-app-native&pullRequest=5050&metric=new_coverage&view=list) [91.2% Coverage](https://sonarcloud.io/component_measures?id=pass-culture_pass-culture-app-native&pullRequest=5050&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=pass-culture_pass-culture-app-native&pullRequest=5050&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=pass-culture_pass-culture-app-native&pullRequest=5050&metric=new_duplicated_lines_density&view=list)
> 
> idea Catch issues before they fail your Quality Gate with our IDE extension sonarlint [SonarLint](https://www.sonarsource.com/products/sonarlint/features/connected-mode/?referrer=sonarcloud-welcome)

Il y a deux issues Sonar. Je ne sais pas si elles sont OK ou non. Soit il faut les traiter, soit les ignorer :)

File: src/features/offer/components/OfferBody/OfferBody.native.test.tsx [hunk 3]
```diff
@@ -149,7 +146,7 @@ const useFeatureFlagSpy = jest.spyOn(useFeatureFlag, 'useFeatureFlag').mockRetur
 
 const onScroll = jest.fn()
 
-const offerId = 1
+const offerId = 146112
```
Comment (aliraiki) [inline src/features/offer/components/OfferBody/OfferBody.native.test.tsx:149]: Est-ce qu'on veut pas utiliser `mockOffer.id` ici ?

File: src/features/offer/components/OfferBody/OfferBody.web.test.tsx [hunk 4]
```diff
@@ -53,7 +53,7 @@ const openURLSpy = jest.spyOn(Linking, 'openURL')
 
 jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag').mockReturnValue(false)
 
-const offerId = 1
+const offerId = 146112
```
Comment (aliraiki) [inline src/features/offer/components/OfferBody/OfferBody.web.test.tsx:56]: De même

File: src/features/offer/components/shareMessagingOffer/VenueMessagingApps.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React, { useCallback } from 'react'
```
Comment (aliraiki) [inline src/features/offer/components/shareMessagingOffer/VenueMessagingApps.tsx:1]: Il faudrait déplacer ce fichier dans la feature `venue`

**Review by aliraiki — APPROVED** [review]
Très clean merci <3

File: src/features/offer/components/shareMessagingOffer/VenueMessagingApps.tsx (hunk 5 above)
Comment (marineshaw) [reply src/features/offer/components/shareMessagingOffer/VenueMessagingApps.tsx:1]: Le truc c'est qu'on a tout mis dans shareMessagingOffer et qu'on a du coup des composants communs que je sais pas ou mettre... T'as des idées ?

Comment (marineshaw) [reply src/features/offer/components/shareMessagingOffer/VenueMessagingApps.tsx:1]: Je peux tout mettre dans la feature share potentiellement ?

Comment (aliraiki) [reply src/features/offer/components/shareMessagingOffer/VenueMessagingApps.tsx:1]: Tu peux mettre dans `share`

**Review by Lucasbeneston — APPROVED** [review]

**Pull request merged**